db.sqlite - compilation fails when trying to assign the last inserted id

### Describe the bug

Hi, i think i hit a bug in compiling ORM code with **sqlite** db

ORM insert works fine when the result is not assigned to another variable i.e. inserted_id
         ```
//address_id := sql db {   // this fails to compile
	sql db {  // this works
		insert address into Address
	} or {
		println("error inserting address")
		panic(err)
	}
```
results in comilation error 'pg_db_last_id' not found

### Reproduction Steps


assign the sql db {insert statement} result to a variable for getting the last inserted id

### Expected Behavior

we should be able to retrieve the ID from the last insert operation using ORM; moreover, pg_DB_last_id  should not be used internally when using sqlite db

### Current Behavior


results in compilation error
================== C compilation error (from tcc): ==============
cc: C:/Users/saasi/AppData/Local/Temp/v_0/scrape_medicine_brands.01JAHMGRX1T75AD40YT050T1A5.tmp.c:41663: warning: implicit declaration of function 'db__pg__DB_last_id'
cc: tcc: error: undefined symbol 'db__pg__DB_last_id'
... (the original output was 30 lines long, and was truncated to 2 lines)
=================================================================
(You can pass `-cg`, or `-show-c-output` as well, to print all the C error messages).
	

### Possible Solution

_No response_

### Additional Information/Context

_No response_

### V version

V 0.4.8 d1b0370

### Environment details (OS name and version, etc.)

windows 10
> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

--- a/vlib/orm/orm_insert_test.v
+++ b/vlib/orm/orm_insert_test.v
@@ -421,3 +421,36 @@ fn test_i64_primary_field_works_with_insertions_of_id_0() {
 	assert users.len == 2
 	// println("${users}")
 }
+
+struct Address {
+	id     i64 @[primary; sql: serial]
+	street string
+	number int
+}
+
+fn test_the_result_of_insert_should_be_the_last_insert_id() {
+	db := sqlite.connect(':memory:')!
+	address := Address{
+		street: 'abc'
+		number: 123
+	}
+	dump(address)
+	sql db {
+		create table Address
+	} or {}
+	aid1 := sql db {
+		insert address into Address
+	} or { panic(err) }
+	dump(aid1)
+	aid2 := sql db {
+		insert address into Address
+	} or { panic(err) }
+	dump(aid2)
+	assert aid2 == 2
+	addresses := sql db {
+		select from Address
+	}!
+	dump(addresses)
+	assert addresses.len == 2
+	assert addresses.all(it.street == 'abc' && it.number == 123)
+}

--- a/vlib/v/gen/c/orm.v
+++ b/vlib/v/gen/c/orm.v
@@ -61,9 +61,7 @@ fn (mut g Gen) sql_insert_expr(node ast.SqlExpr) {
 		node.or_expr)
 
 	g.write(left)
-	g.write('db__pg__DB_last_id(')
-	g.expr(node.db_expr)
-	g.write(');')
+	g.write('orm__Connection_name_table[${connection_var_name}._typ]._method_last_id(${connection_var_name}._object)')
 }
 
 // sql_stmt writes C code that calls ORM functions for

